week-4/InClass/B-objects-get-set/exercise-2.js

```diff
@@ -5,14 +5,14 @@
 */
 
 let phone = {
-    brand: 'iPhone,
-    model 'iPhone X'
+    brand: 'iPhone',
+    model: 'iPhone X',
     launchYear: 2017,
-    is Unlocked: true
-;
+    isUnlocked: true
+};
 
-let phoneBrand = phone.bbrand;
-let phoneLaunchYear = phone[launchYear];
+let phoneBrand = phone.brand;
+let phoneLaunchYear = (phone['launchYear']);

```

Comment:
Well spotted syntax errors. However, why do you need () around `phone['launchYear']` ?